src/app/components/forms/post/post.styl

```diff
@@ -105,12 +105,19 @@
 		margin-bottom: 4px
 
 		&-leave-active
```

Comment:
You can probably just use the no-animate-leave class on the element.

Reply:
Changed to using `no-animate-leave` and `anim-fade-in-enlarge` classes on incomplete, add, and normal community pills in this context.